File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int p_cnt = 0;
    int y_cnt = 0;

    for (int i=0; i<s.length(); i++) {
        if (s[i] == 'p' || s[i] == 'P') p_cnt += 1;
        if (s[i] == 'y' || s[i] == 'Y') y_cnt += 1;
    }

    if (p_cnt == y_cnt) return true;
    else return false;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;

    let p_n = 0;
    let y_n = 0;
    
    for (let i=0; i<s.length; i++) {
        if (s[i] == 'p' || s[i] == 'P') p_n+=1;
        if (s[i] == 'y' || s[i] =='Y') y_n+=1;
    }
    
    if (p_n == y_n) return true;
    else return false;
}